examples/Cargo.toml

```diff
@@ -224,6 +225,11 @@ required-features = ["openssl-tls", "rustls-tls"]
 name = "custom_client_trace"
 path = "custom_client_trace.rs"
 
+[[example]]
+name = "cert_check"
+path = "cert_check.rs"
+required-features = ["unstable-client"]
```

Comment:
personally i don't think you need to add this required feature (and the feature) in here because both tls stacks implicitly add it